frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -100,6 +99,24 @@ const defaultFilterValues: IPropertyFilter = {
   maxNetBookValue: '',
 };
 
+export const PropertyPopUpContext = React.createContext({
+  propertyInfo: null,
+  setPropertyInfo: noop,
+  propertyTypeID: null,
+  setPropertyTypeID: noop,
+});
+const PropertyPopUpContextProvider: React.FC = ({ children }) => {
+  const [propertyInfo, setPropertyInfo] = React.useState(null);
+  const [propertyTypeID, setPropertyTypeID] = React.useState(null);
+  return (
+    <PropertyPopUpContext.Provider
+      value={{ propertyInfo, setPropertyInfo, propertyTypeID, setPropertyTypeID }}
+    >
+      {children}
+    </PropertyPopUpContext.Provider>
+  );
+};
+
```

Comment:
I think I was told to put this definition somewhere else but I've forgotten where

Reply:
Ideally in it's own file, trying to keep the map code from getting more bloated.